src/widget/translator.cpp

```diff
@@ -72,6 +73,15 @@ void Translator::translate()
         QCoreApplication::installTranslator(translator);
     }
 
+    // After change language from rtl to ltr not always restored layout direction
```

Comment:
`After the language is changed from RTL to LTR, the layout direction isn't always restored` ?